### ISSUE TITLE
Problem Tab is all corrupted with Typescript Problems

### ISSUE THREAD

**Issue by Bene-Graham** [Microsoft/vscode#56371]
![problemwindowissues](https://user-images.githubusercontent.com/35272170/44098850-3f965910-9faf-11e8-99a9-9b66cbe3c2c0.png)

The other thing that happens when it is in this state is the scroll in the problem window is all wired. It is almost like the first half is floating on top.

- VSCode Version: 1.26
- OS Version: OSX High Sierra

Steps to Reproduce:

Unfortunately I can't find the exact steps to reproduce, but it has happened a few times this morning. From I what I can tell it seems occurs when either a large block of problem code is commented out or using a couple of TypeScript fixers back to back. 

This was not happening in the previous version: 1.25.1.

**Comment by Bene-Graham** [comment]
Just an update. I tried disabling the TsLint extension and the problem still exists.

**Comment by mihe** [comment]
Been having this problem since updating to 1.26.0 as well. Like previously mentioned it seems difficult to reproduce. I've tried running with `--disable-extensions` and the issue still presents itself after a while.

![fc5b7fbcee](https://user-images.githubusercontent.com/4884246/44099892-f775ecb0-9fe3-11e8-87bc-914f0189f4eb.gif)

OS Version: Windows 10.0.16299

**Comment by shir** [comment]
Have similar problem for ruby files (using [Ruby extenstion](https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby)). And for typescript files too.

**Comment by shawnlewis** [comment]
Downgrading to 1.25.1 fixes it. You can get it here: https://code.visualstudio.com/updates/v1_25. Disable automatic updates by putting "update.channel": "none" in your user settings.

**Comment by DanTup** [comment]
Seeing same thing in Dart. I'm seeing stale errors, and clicking them doesn't navigate anywhere, and the same scrollbar that does nothing (which shouldn't be displayed, because the list is short).

**Comment by dbaeumer** [comment]
@sandy081 since this happens with many languages this looks like a general problem with the view. Might be even Electron 2.0 related due to the scrolling issues.

**Comment by mihe** [comment]
Related to #56317 by the looks of it.

**Comment by sandy081** [comment]
Can anyone help in providing the steps to reproduce this (possibly with some sample file)?

**Comment by DanTup** [comment]
Interestingly I haven't seen it on my Windows PC today (it was MacBook I saw it on). When I next see it (either here, or on the MB I'll see if I can do it on-demand and provide any details I can).

**Comment by dchw** [comment]
I am on macOS Mojave, and this project produces the issue. Its a quick and dirty one I made with `dotnet` commands exclusively.
[testProj.zip](https://github.com/Microsoft/vscode/files/2291032/testProj.zip)

**Comment by DanTup** [comment]
Well, I saw it on Windows:

![win](https://user-images.githubusercontent.com/1078012/44161487-63482f80-a0b5-11e8-8da5-2b4c4b4f2cfb.png)

Same thing - the top entries have lost their file (they should be in the `interpreter.dart` section which is further down.

I reviewed the language server logs, and the errors all look normal to me. They all came through with the correct path (and the casing is all correct, etc.). I can't figure out how to reliably repro though, just seems to happen after a while.

I did find lots of errors in the dev console ([full log](https://github.com/Microsoft/vscode/files/2291042/console.txt)):

Here's the unique ones:

```
ERR An unknown error occurred. Please consult the log for more details.

ERR Cannot read property 'hasFixes' of null: TypeError: Cannot read property 'hasFixes' of null
    at t [as constructor] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3575:721)
    at t.create (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:111:339)
    at e._createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1378:703)
    at e.createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1377:959)
    at e.renderMarkerElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:926)
    at e.renderElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:218)
    at e.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:99)
    at t.refreshViewItem (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:967:916)
    at t.onRefreshItem (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:967:754)
    at t.e.onRemoveItems (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:344:585)
    at t.onItemChildrenRefreshed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:964:920)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:174:517
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:463
    at n.Class.derive._creator.then (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:366)
    at s (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:416)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:795
    at new n.Class.derive._oncancel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:625)
    at e.run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:669)
    at t.e.refreshChildren (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:497)
    at t.e.doRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:751)
    at t.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:893)
    at e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:335:201)
    at i.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:981:186)
    at t.refreshPanel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4439:977)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4443:303
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:402
    at Object.g [as _notify] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:151:729)
    at Object.enter (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:155:61)
    at n.Class.derive._oncancel._run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:902)
    at n.Class.derive._oncancel._completed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:343)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:461
/C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1398

ERR Cannot read property 'top' of undefined: TypeError: Cannot read property 'top' of undefined
    at t.e.onRefreshItems (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:345:1)
    at t.e.onRefreshItemSet (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:344:763)
    at t.onItemsRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:965:469)
    at t.onItemRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:965:393)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:174:517
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at e.doRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:735)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:285
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:329:178
    at t.e.forEachChild (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:329:78)
    at t.e.mapEachChild (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:329:146)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:251
    at n.Class.define.cancel.then (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:366)
    at s (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:326:629)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:795
    at new n.Class.derive._oncancel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:625)
    at e.run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:669)
    at t.e.refreshChildren (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:497)
    at t.e.doRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:751)
    at t.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:893)
    at e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:335:201)
    at i.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:981:186)
    at t.refreshPanel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4439:977)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4443:303
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:402
    at Object.g [as _notify] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:151:729)
    at Object.enter (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:155:61)
    at n.Class.derive._oncancel._run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:902)
    at n.Class.derive._oncancel._completed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:343)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:461

ERR Cannot read property 'id' of null: TypeError: Cannot read property 'id' of null
    at t.e.onInsertItems (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:344:22)
    at t.onItemChildrenRefreshed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:965:248)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:174:517
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:463
    at n.Class.derive._creator.then (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:366)
    at s (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:416)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:795
    at new n.Class.derive._oncancel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:625)
    at e.run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:669)
    at t.e.refreshChildren (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:497)
    at t.e.doRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:751)
    at t.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:893)
    at e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:335:201)
    at i.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:981:186)
    at t.refreshPanel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4439:977)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4443:303
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:402
    at Object.g [as _notify] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:151:729)
    at Object.enter (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:155:61)
    at n.Class.derive._oncancel._run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:902)
    at n.Class.derive._oncancel._completed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:343)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:461

view item doesnt exist
e.onRemoveItems @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:344

view item doesnt exist
e.onInsertItems @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:343
```

**Comment by sandy081** [comment]
@DanTup Thanks for the logs. I think I understand the issue now.

**Comment by sandy081** [comment]
There is an exception while rendering a problem entry and hence it is not getting removed and remains. This is also the reason why the interaction is also broken.

This exception is caused due to the timing issue. Just before the problem entry is rendered, its parent is removed from the tree. While the problem entry is being rendered, I am accessing its parent using tree navigator (assuming that parent should exist always) to check for the quick fixes. Since the parent is removed already it does not exist and causing this exception.

Fix is to not to access the parent through tree navigator, instead from the model which will assure it always exists.

**Comment by joaomoreno** [comment]
@sandy081 I've wrapped tree `renderElement` and `renderTemplate` calls with `try catch` blocks, to prevent the tree from breaking when errors happen within those calls in the future. :+1: Pushed to master, not for the recovery.

**Comment by Domvel** [comment]
Related  to #56317 Some comments of #56317 also descripes this error. In my case a ionic angular project in `1.26.0` with tslint by egamma and a trailing comma looks so:

![44097281-add21422-9fdd-11e8-8b0a-9504265bcc6b](https://user-images.githubusercontent.com/35452565/44196818-45bea880-a13d-11e8-83d8-063ddb654f2b.png)
(bug / glitch)

on `1.25.1` so:
![44098887-a4a764b6-9fe1-11e8-85ec-a5f955d7bfe6](https://user-images.githubusercontent.com/35452565/44196828-4bb48980-a13d-11e8-83f7-1353da37cef9.png)
(correct)

I don't know the steps to reproduce it. e.g. after restart of vs code the bug disapears.

**Comment by aeschli** [comment]
Code reviewed https://github.com/Microsoft/vscode/commit/26bd850ab93d2073b9a3870db1b78af14d9e3f5a

**Comment by aeschli** [comment]
Verified in 1.26.1-stable
Commit: 49e7ddb857cda50c123f37f39cb1ef8894bf53bb

**Comment by Domvel** [comment]
@sandy081 Sorry, I should do this for an issue next time. Currently, the bug no longer appears. Because I want log him. 😄 Typical debugging issue. I'll work on `1.26.0` and hope the bug appears and post the log here. But whatever ... The bug is fixed in `1.26.1` right?

**Comment by Domvel** [comment]
@sandy081 One second late of my post above the bug appeared. Easy trick to reproduce him. e.g. trailing commas (if tslint is enabled for this). Just typ a lot of commas and remove it again. At last add once again. Voila, here it is.

![image](https://user-images.githubusercontent.com/35452565/44208690-6ef13000-a161-11e8-9717-f6194b3cbcbc.png)

```text
  ERR Cannot read property 'hasFixes' of null: TypeError: Cannot read property 'hasFixes' of null
    at t [as constructor] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3575:721)
    at t.create (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:111:339)
    at e._createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1378:703)
    at e.createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1377:959)
    at e.renderMarkerElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:926)
    at e.renderElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:218)
    at e.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:99)
    at t.refreshViewItem (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:967:916)
    at t.onRefreshItem (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:967:754)
    at t.e.onRemoveItems (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:344:585)
    at t.onItemChildrenRefreshed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:964:920)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:552)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:174:517
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:463
    at n.Class.derive._creator.then (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:366)
    at s (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:416)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:795
    at new n.Class.derive._oncancel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:625)
    at e.run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:319:669)
    at t.e.refreshChildren (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:497)
    at t.e.doRefresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:751)
    at t.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:327:893)
    at e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:335:201)
    at i.e.refresh (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:981:186)
    at t.refreshPanel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4439:977)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4443:303
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:402
    at Object.g [as _notify] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:151:729)
    at Object.enter (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:155:61)
    at n.Class.derive._oncancel._run (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:902)
    at n.Class.derive._oncancel._completed (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:156:343)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:195:461
log.ts:171   ERR Cannot read property 'hasFixes' of null: TypeError: Cannot read property 'hasFixes' of null
    at t [as constructor] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3575:721)
    at t.create (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:111:339)
    at e._createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1378:703)
    at e.createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1377:959)
    at e.renderMarkerElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:926)
    at e.renderElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:218)
    at e.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:99)
    at e.insertInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:917)
    at t.insertItemInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:976:302)
    at t.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:959:133)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:954:429
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:392:566
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at t._setState (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:255:372)
    at t.setScrollPositionNow (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:253:889)
    at t._onMouseWheel (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:396:706)
    at HTMLDivElement.n (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:395:652)
log.ts:171   ERR Cannot read property 'hasFixes' of null: TypeError: Cannot read property 'hasFixes' of null
    at t [as constructor] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3575:721)
    at t.create (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:111:339)
    at e._createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1378:703)
    at e.createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1377:959)
    at e.renderMarkerElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:926)
    at e.renderElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:218)
    at e.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:99)
    at e.insertInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:917)
    at t.insertItemInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:976:302)
    at t.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:959:133)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:954:429
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:392:566
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at t._setState (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:255:372)
    at t.setScrollDimensions (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:253:438)
    at t.setScrollDimensions (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:394:818)
    at t.set [as viewHeight] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:962:347)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:958:653)
    at i.e.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:980:699)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4438:265)
    at c.t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4916:289)
    at c.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4966:150)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4499:832)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4492:290
    at n.Class.define.cancel.done (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:234)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4492:265
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:387:138
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at r (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:181:588)
log.ts:171   ERR Cannot read property 'hasFixes' of null: TypeError: Cannot read property 'hasFixes' of null
    at t [as constructor] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:3575:721)
    at t.create (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:111:339)
    at e._createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1378:703)
    at e.createInstance (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:1377:959)
    at e.renderMarkerElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:926)
    at e.renderElement (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4434:218)
    at e.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:99)
    at e.insertInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:951:917)
    at t.insertItemInDOM (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:976:302)
    at t.render (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:959:272)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:954:429
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:392:566
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at t._setState (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:255:372)
    at t.setScrollDimensions (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:253:438)
    at t.setScrollDimensions (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:394:818)
    at t.set [as viewHeight] (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:962:347)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:958:653)
    at i.e.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:980:699)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4438:265)
    at c.t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4916:289)
    at c.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4966:150)
    at t.layout (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4499:832)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4492:290
    at n.Class.define.cancel.done (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:158:234)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4492:265
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:387:138
    at e.fire (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:172:532)
    at r (file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:181:588)
```

**Comment by sandy081** [comment]
@Domvel This will be fixed in 1.26.1
